snooty/parser.py

```diff
@@ -616,7 +618,19 @@ def handle_directive(
                     self.validate_list_table(bullets, expected_num_columns)
 
         elif name == "openapi":
+            # Parsing should be done by the OpenAPI renderer on the frontend by default
+            snooty_parse = options.get("snooty-parse", False)
+
             if argument_text is None:
+                # Check if argument is a url instead
+                url_argument = None
+                try:
+                    url_argument = argument[0].children[0].value
+                except:
+                    pass
+                if not (url_argument is None or snooty_parse):
+                    doc.options["isUrl"] = "True"
```

Comment:
I added that flag so that the [frontend could determine which value to use](https://github.com/mongodb/snooty/pull/338/files#diff-1aef327520208d3233342113f027cd9a62bfbbb52295c9e9b5bb138be5aa3b3dR20-R24) (filepath or url) and what to do with said value before passing it to Redoc.

I could change the frontend logic so that it doesn't have to rely on this flag!

Reply:
I think it may be preferable to change the front end, mainly because we don't particularly embrace a pattern of setting options internally in the parser. Generally, I think one expect to find all of the options enumerated in the `rstspec.toml`, so we may as well not change tack!